2.6/run-mongod.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# For SCL enablement
+source /var/lib/mongodb/common.sh
+
+set -eu
+
+# Data directory where MongoDB database files live. The data subdirectory is here
+# because .bashrc and mongodb.conf both live in /var/lib/mongodb/ and we don't want a
```

Comment:
there's no `.bashrc` file in this repo nor in `/var/lib/mongodb/`, this is a historical leftover.